docs/HowTo/Configure/Consensus/Configuring-Raft.md

```diff
@@ -28,9 +28,11 @@ and issue `raft.promoteToPeer(raftId)`.
 The enode ID must include a `raftport` querystring parameter like the [enode IDs in the `static-nodes.json`
 file](#configuring-raft-consensus).
 
-The `addPeer` and `addLearner` calls allocate and return a Raft ID not already in use. After `addPeer`
-or `addLearner`, start the new GoQuorum node with the `--raftjoinexisting RAFTID` command line option
-in addition to `--raft`. Replace `RAFTID` with the Raft ID returned by `addPeer` or `addLearner`.
+The `addPeer` and `addLearner` calls allocate and return a Raft ID not already in use.
+After `addPeer` or `addLearner`, start the new GoQuorum node with the
+[`--raftjoinexisting`](../../../Reference/CLI-Syntax.md#raftjoinexisting) command line option in addition to
```

Comment:
"start the new GoQuorum node with the `--raftjoinexisting` and `--raft` command line options."